server/lib/util/logger.js

```diff
@@ -1,35 +1,49 @@
 const log = require('pino')()
```

Comment:
we could use logger from core here, its the same thing.

Reply:
-100.. Core logger is probably temporary there. For libraries people tend to use debug package